modules/jupyter-widget/src/playground/create-deck.js

```diff
@@ -113,14 +113,14 @@ function createStandaloneFromProvider({
   // Common deck.gl props for all basemaos
   const handlers = handleEvent
     ? {
-        onClick: info => handleEvent('click', info),
-        onHover: info => handleEvent('hover', info),
-        onResize: size => handleEvent('resize', size),
+        onClick: info => handleEvent('deck-click-event', info),
+        onHover: info => handleEvent('deck-hover-event', info),
+        onResize: size => handleEvent('deck-resize-event', size),
         onViewStateChange: ({viewState, interactionState, oldViewState}) => {
```

Comment:
I realize now that I added the debouncing logic for this [in pydeck](https://github.com/visgl/deck.gl/blob/master/bindings/pydeck/pydeck/widget/debounce.py) but we may want debouncing on the client instead, given that we'll have to implement it in each binding.